harvard-lts/CURIOSity
# Update View Files: part 1

# Summary

The theming for the following pages have been updated based on the mockups [that can be seen here](https://www.sketch.com/s/39eb3859-75e8-4add-8498-6cbf2934d73c/a/25a1d2e)
- Landing Page
- Exhibit Home Page
- Exhibit Search Results Page
- Exhibit subpages (Browse, About, Feature pages)

## Related Ticket
https://github.com/harvard-lts/CURIOSity/issues/10 - update view files
https://github.com/harvard-lts/CURIOSity/issues/23 - collection landing page

## Related Github MR's

To see more details about this work, feel free to check these Gitlab MRs.

https://gitlab.com/notch8/harvard-curiosity/-/merge_requests/28
https://gitlab.com/notch8/harvard-curiosity/-/merge_requests/27
https://gitlab.com/notch8/harvard-curiosity/-/merge_requests/26

## How should this be tested?
- Check on [staging](https://staging-harvard-curiosity.notch8.cloud/) to make sure the app looks consistent to the mockups
- You may need to fill out the appearance/site title/subtitle/description etc, and create an example exhibit on staging that mimics real exhibit data to see all the pages.
- **Note:** the harvard logo in the top right corner might be broken on staging. I think this is an issue due to SVGs in the rails asset pipeline. [I have created a separate ticket to fix these.](https://github.com/harvard-lts/CURIOSity/issues/75)

## Interested parties
@VanessaVenti @enriquediaz
